src/tribler-core/tribler_core/upgrade/tests/test_config_upgrade_70_71.py

```diff
@@ -33,15 +33,15 @@ def test_read_test_libtribler_conf(self):
         """
         Test upgrading a libtribler configuration from 7.0 to 7.1
         """
-        os.environ['TSTATEDIR'] = self.session_base_dir.to_text()
+        os.environ['TSTATEDIR'] = str(self.session_base_dir)
         old_config = RawConfigParser()
-        old_config.read(self.CONFIG_PATH / "libtribler70.conf")
+        old_config.read(pathlib.Path(self.CONFIG_PATH, "libtribler70.conf"))
         new_config = TriblerConfig()
         result_config = add_libtribler_config(new_config, old_config)
         self.assertEqual(result_config.get_tunnel_community_socks5_listen_ports(), [1, 2, 3, 4, 5, 6])
         self.assertEqual(result_config.get_anon_proxy_settings(), (2, ("127.0.0.1", [5, 4, 3, 2, 1]), ''))
         self.assertEqual(result_config.get_credit_mining_sources(), ['source1', 'source2'])
-        self.assertEqual(result_config.get_log_dir(), path_util.Path('/a/b/c').absolute())
```

Comment:
What's this `absolute` method? Is it suppose to be `resolve` or `normpath` or something? For `pathlib.Path` and `path_util.Path` respectively.

https://github.com/Tribler/tribler/blob/devel/src/tribler-core/tribler_core/utilities/path_util.py
https://docs.python.org/3/library/pathlib.html#pathlib.Path.resolve

Reply:
absolute means resolve a relative path like `../foo/bar/` to something like `/tmp/foo/bar`. It should be used sparingly, mostly to save system-dependent paths.

Reply:
Ah. Gotcha. Is it alright in this case to replace `absolute` with `resolve`?

Reply:
Yes. But try to remove it at all.

Reply:
Remove `test_read_test_tribler_conf`, or the specific assertion only?